### PR TITLE
Error for when users run Mako and it's already running.

### PR DESCRIPTION
**Pull request by qvm1Wak**
This tripped me up for a second, so maybe the message will help the next person.

### CODE REVIEW

File: dbus/dbus.c [hunk 1]
```diff
@@ -31,7 +31,7 @@ bool init_dbus(struct mako_state *state) {
 
 	ret = sd_bus_request_name(state->bus, service_name, 0);
 	if (ret < 0) {
-		fprintf(stderr, "Failed to acquire service name: %s\n", strerror(-ret));
+		fprintf(stderr, "Failed to acquire service name: %s. Is Mako already running?\n", strerror(-ret));
```
Comment (emersion) [inline dbus/dbus.c]: Maybe we can check the value of `ret` and only print the new message in case it's already running?

**Comment by vilhalmer** [comment]
I like this, but it should probably be "is another notification daemon running?" because it will show up if anything owns the name `org.freedesktop.Notifications`, not just mako.

File: dbus/dbus.c [hunk 2]
```diff
@@ -31,7 +31,11 @@ bool init_dbus(struct mako_state *state) {
 
 	ret = sd_bus_request_name(state->bus, service_name, 0);
 	if (ret < 0) {
-		fprintf(stderr, "Failed to acquire service name: %s. Is Mako already running?\n", strerror(-ret));
+                if (ret == -EEXIST) {
```
Comment (emersion) [inline dbus/dbus.c]: Style: we use tabs

File: dbus/dbus.c [hunk 3]
```diff
@@ -31,7 +31,11 @@ bool init_dbus(struct mako_state *state) {
...
+                        fprintf(stderr, "Failed to acquire service name: %s. Is Mako already running?\n", strerror(-ret));
```
Comment (emersion) [inline dbus/dbus.c]: We can probably always print the existing log line, but print an additional one after if `ret == -EEXIST`, saying the another notification daemon is already running.

**Comment by emersion** [comment]
Anyway, this looks good to me as-is

**Comment by emersion** [comment]
Thanks!

**Pull request merged**